test/unittest/tests/block/test_extent_list_insert_wo_block.cpp

```diff
@@ -0,0 +1,197 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * block_ext.c: [extent_list2] Test extent list functions part 2. (More to come.)
+ *
+ * Test insert functions without block: __block_ext_insert, and __block_off_insert.
+ */
+
+/* Choose one */
+#define DEBUG /* Print debugging output */
+//#undef DEBUG
+
+#include <algorithm>
+#include <memory>
+
+#include <catch2/catch.hpp>
+
+#include "test_util.h"
+#include "../utils.h"
+#include "../utils_extlist.h"
+#include "../wrappers/mock_session.h"
+#include "wt_internal.h"
+
+using namespace utils;
+
+/*!
+ * To sort off_size by _off and _size
+ */
+struct {
+    bool
+    operator()(const off_size &left, const off_size &right)
```

Comment:
That's a nice use of a comparison function for `std::sort`. In this case, the `operator()` can also be made const.